### PR TITLE
Update Repeticiones (TWP15) lecture

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#31.
**Note**: I didn't focus on changing redaction nor grammar. 

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Reviewers assigned (all peers & at least 1 mentor)

### CODE REVIEW

File: _sources/lectures/TWP15.rst [hunk 1]
```diff
@@ -150,68 +150,80 @@ Contadores
...
+       x = x + 2
 
 
-Contadores
-==========
+Algunos ejercicios
+------------------
 
 
+.. activecode:: ac_l15_4
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP15.rst:168]: You could add a playwright test to this activecode block

**Review by angelasofiaremolinagutierrez — COMMENTED** [review]
Everything looks ok

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
LGTM!

**Review by y-martinez — APPROVED** [review]
LGTM 🚀

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

**Pull request merged**